avs_client/refreshtoken/handlers.py

```diff
@@ -67,4 +67,7 @@ def handle_callback(self):
             self.end_headers()
             refresh_token = response.json()['refresh_token']
             self.wfile.write(bytes('refresh_token: ' + refresh_token, 'utf8'))
+            self.wfile.write(bytes('\n\n ', 'utf8'))
+            access_token = response.json()['access_token']
+            self.wfile.write(bytes('access_token: ' + access_token, 'utf8'))
```

Comment:
Why is displaying the access token helpful here?

We could simply display on screen return everything in Amazons's response if that would be helpful?

let me know the usecase :)